appstudio-utils/util-scripts/lib/fetch.sh

```diff
@@ -16,6 +16,8 @@ EC_WORK_DIR=${EC_WORK_DIR:-/tmp/ecwork}
 
 DATA_DIR=${DATA_DIR:-"$EC_WORK_DIR/data"}
 POLICIES_DIR=${POLICIES_DIR:-"$EC_WORK_DIR/policies"}
+mkdir -p $DATA_DIR
+mkdir -p $POLICIES_DIR
```

Comment:
We happen to know there's a mkdir in the git clone method, so technically we don't need to create $POLICIES_DIR but I think it's best to leave this here anyway to be consistent and to reduce our 'wtf per minute' score.